### ISSUE TITLE
Connecting To Televantage Sever

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sonuarya/csipsimple#1102]
```
What steps will reproduce the problem?
1.Input Settings In Phone 
   account id: <sip:user@xxx.xxx.xxx.xxx:5060>
   Reg URI: sip:xxx.xxx.xxx.xxx:5060
   Proxy:  sip:xxx.xxx.xxx.xxx:5060
2.Obain Binding from Server
3.Try to initiate call

What is the expected result? I would expect that the binding would be on port 
for the contact URI to be 5060.     What do you see instead?
The resut that I get is sip:user@xxx.xxx.xxx.xxx:37706;ob

What version of the product are you using? On what operating system?
Android r944

Please provide any additional information below.

With Televantage Server when you connect and register your binding you must 
have a contact uri with port 5060. If you dont have port 5060 and you initiate 
a call the android will ring but once you connect the call the call is dropped. 
We have had the same issue with Aastra SIP phone and we fixed by forcing the 
port 5060. With csipsimple even though we specify the port in the settings we 
are still not getting the binding with the port 5060. Always in the port range 
from 33000-52000.

So to recap. We have no problem getting a registration with the server, we have 
a problem with incomming calls dropping when the call is answered.   


```

Original issue reported on code.google.com by `mydog...@gmail.com` on 24 Jun 2011 at 3:40
